ui/selectors/transactions.test.js

```diff
@@ -254,7 +254,7 @@ describe('Transaction Selectors', () => {
   });
 
   describe('nonceSortedTransactionsSelector', () => {
-    it('returns transaction group nonce sorted tx from from selectedTxList wit', () => {
```

Comment:
Lol, there's a second problem on this line.  Clearly that was supposed to say `from selectedTxList with _______` but it got cut off.  Does anyone know what this was supposed to say?

Reply:
No, your modifications are fine, but I guess we should try to figure out what that line was supposed to say.  I looked at Git history and I think it was always cut off.

Reply:
I think actually let's just remove the word ` wit`